Adding files for VMware ESXi BYOI for HPE Private Cloud Enterprise - Bare Metal

nareshbhalala opened the pull request:
Adding files for VMware ESXi Bring Your Own Image (BYOI) for HPE Private Cloud Enterprise - Bare Metal

```
./LICENSE
./CONTRIBUTING.md
./README.md
./.gitignore
./*.sh
./*.template
```

nareshbhalala committed:
Adding files for VMware ESXi BYOI for HPE Private Cloud Enterprise - Bare Metal

haybill reviewed (approved):
Looks Good To Me

neilgierman commented on README.md:
```diff
@@ -1 +1,549 @@
+> VMware Customer Connect simplifies the management of free trials, downloads, and support.
+
+This ESXi recipe has been successfully tested with the following version of ESXi:
+* ESXi 7.0-U3p
```
Haven't we also tested with ESXi 7.0-U3o?

nareshbhalala replied:
@neilgierman  , @haybill:

yes, now based on the discussion a few minutes before, we need to update this comment as below:

Either **generic statement without specifying build o or p**:
```
This ESXi recipe has been successfully tested with the following version of ESXi:
* ESXi 7.0-U3
```
OR **specific to working version**:
```
This ESXi recipe has been successfully tested with the following version of ESXi:
* ESXi 7.0-U3o
```
OR **listing all**:
```
This ESXi recipe has been successfully tested with the following version of ESXi:
* ESXi 7.0-U3o
* ESXi 7.0-U3p
```

Please suggest.

neilgierman replied:
I would list both o and p because we did test this recipe in both those versions. We have not tested this recipe on every other version of U3.

nareshbhalala replied:
Than kyou @neilgierman 

Changed to:
```
This ESXi recipe has been successfully tested with the following version of ESXi:
* ESXi 7.0-U3o
* ESXi 7.0-U3p
```

nareshbhalala committed:
Update README.md

Adding the entry "ESXi 7.0-U3o" under the list of ESXi recipe has been successfully tested.

neilgierman reviewed (approved):
Looks good to me now.
